CMakeLists.txt

```diff
@@ -805,6 +812,10 @@ target_include_directories(${OFFSETS_LIB} PRIVATE
   kernel/include
   ${ARCH_DIR}/${ARCH}/include
   )
+
+# Make sure that LTO will never be enabled when compiling offsets.c
+set_source_files_properties(${OFFSETS_C_PATH} PROPERTIES COMPILE_OPTIONS -fno-lto)
```

Comment:
this may not work properly accross non-gcc based toolchains.

Reply:
This one requires some checks if this is even required.

Reply:
Yes - it is required. Added compiler option prohibit_lto.